server/docker.yaml

```diff
@@ -0,0 +1,6 @@
+docker:
```

Comment:
The comment version was just an example. The full version would be something like this:
```
docker:
    command: docker
    # init_command: ?
    start_command: \{command} run --name {name} -d {image}
    stop_command: \{command} stop {name}
    pull_command: \{command} pull {image}

```

You may need to add a name as well to the server config so docker can stop it by name.

Reply:
Updated the docker.yaml file